ContactPage.php

```diff
@@ -42,3 +42,23 @@
 		'RLStyleModules' => array(),
 	);
 }
+
+if ( $wgDBname == 'fablabesdswiki' ) {
+	$wgContactConfig['default'] = array(
+		'RecipientUser' => 'Contact COOP FabLab ESDS',
+		'SenderEmail' => $wgPasswordSender,
+		'SenderName' => 'Ne pas répondre FabLab ESDS',
```

Comment:
It implies a 'Do Not Reply' therefore it is fine, it's only a conical name and has no impact.